lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -0,0 +1,83 @@
+module Runners
+  class Processor::MetricsFileInfo < Processor
+    Schema = _ = StrongJSON.new do
+      # @type self: SchemaClass
+
+      let :runner_config, Schema::BaseConfig.base
+      let :issue, object(
+          line_of_code: integer?,
+          last_commit_datetime: string
+      )
+    end
```

Comment:
[nitpick] inconsistent indentation

```suggestion
      let :issue, object(
        line_of_code: integer?,
        last_commit_datetime: string
      )
    end
```

Reply:
Thx. I've fixed my editor's settings.

Reply:
fix: 989b6dc31fe7409e407f0dbd148571321b82eece